cli/cmd/devnet.go

```diff
@@ -171,6 +171,7 @@ func devnetDefinition(ctx context.Context) (app.Definition, error) {
 
 	defCfg := app.DefaultDefinitionConfig(ctx)
 	defCfg.ManifestFile = manifestFile
+	defCfg.GraphQLURL = "http://explorer_graphql:8080/query"
```

Comment:
rather add this as the "default" GraphQLURL [here](https://github.com/omni-network/omni/blob/main/e2e/app/definition.go#L59).